### PR TITLE
added: productpage

### PR DESCRIPTION
**Pull request by sharmayatendra**
# Product Page

## A main product section where it shows all the products.

image

### CODE REVIEW

**Review by vikrantkode — COMMENTED** [review]
-Looks good to me . Ready to merge

File: Pages/Productpage/productpage.css [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    display: flex;
+}
+
+.align-center {
+    align-items: center;
+}
+
+.space-between {
+    justify-content: space-between;
+}
```
Comment (mohitdhatrak) [inline Pages/Productpage/productpage.css:39]: if you plan to use these classes in all files, maybe make a separate CSS file for such blocks!

File: Pages/Productpage/productpage.css [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    --body-bgcolor: rgb(230, 230, 230);
+}
+
+body {
+    background-color: var(--body-bgcolor);
+}
+
+.container {
+    display: grid;
+    gap: 20px;
```
Comment (yashpatel1797) [inline Pages/Productpage/productpage.css]: use 'rem' here as well

**Review by yashpatel1797 — APPROVED** [review]
Looks good to merge.

**Review by mohitdhatrak — COMMENTED** [review]
Nice use of grids yatendra! Good to merge, just a suggestion, use cursor pointer for options in the filter section!

File: Pages/Productpage/productpage.css (hunk 1 above)
Comment (sharmayatendra) [reply Pages/Productpage/productpage.css:39]: sidebar-nav here is only for the product page that's why I have deliberately written here. BTW thanks for the suggestion will keep that in mind.

File: Pages/Productpage/productpage.css (hunk 2 above)
Comment (sharmayatendra) [reply Pages/Productpage/productpage.css]: done with changes

**Pull request merged**